Bottom: 675e084856472d72da1b47616e7339190cd4f368
Top:    8553590903637a495e2509274c82cf289d470ce0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:21 +0000

Invalidate board contour when a via / pin is altered

The drill-holes of vias and pins can affect the board contour


---

diff --git a/src/polygon.c b/src/polygon.c
index c1f78c1..a9b0f29 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1812,6 +1812,10 @@ ClearFromPolygon (DataType * Data, int type, void *ptr1, void *ptr2)
           strcmp (layer->Name, "route") == 0)
         Data->outline_valid = false;
     }
+  else if (type == PIN_TYPE || type == VIA_TYPE)
+    {
+        Data->outline_valid = false;
+    }
 
   if (type == POLYGON_TYPE)
     InitClip (PCB->Data, (LayerType *) ptr1, (PolygonType *) ptr2);
